data/boundaries/citycountystate/index.html

```diff
@@ -88,6 +88,7 @@ <h5 id="MunicipalBoundaries-updates">Updates</h5>
         </div>
         <div class="panel-content">Updates to this dataset are published at a monthly interval or as needed.</p>
 <p>Recent update summaries:<br />
+<a href="{{site.baseurl}}{% post_url  2016-05-16-first-quarter-updates-and-beyond.md %}">May 16, 2016</a><br />
```

Comment:
you don't need the `.md` extension on the link and watch out for the double white space you have after `post_url`